packages/strapi-plugin-i18n/services/content-types.js

```diff
@@ -83,12 +83,21 @@ const isLocalizedAttribute = (model, attributeName) => {
   const attribute = model.attributes[attributeName];
 
   return (
-    isLocalized(attribute) ||
+    hasLocalizedOption(attribute) ||
     (isRelationalAttribute(attribute) && !isMediaAttribute(attribute)) ||
     isTypedAttribute(attribute, 'uid')
   );
 };
 
+/**
+ * Returns whether a model is localized or not
+ * @param {*} model
+ * @returns
+ */
+const isLocalizedModel = model => {
```

Comment:
if you use hasLocalizedOption internaly then you could still export isLocalized as this is the content-types service & not an attributes service